lib/omnibus/packagers/ips.rb

```diff
@@ -214,17 +214,32 @@ def write_transform_file
     end
 
     #
+    # The name of the project specific template if it exists
+    # The resource exists locally. For example for project omnibus-toolchain
+    # resource_path("#{safe_base_package_name}-symlinks.erb") #=>
+    # {"/path/to/omnibus-toolchain/resources/omnibus-toolchain/ips/omnibus-toolchain-symlinks.erb"}
+    # OR {"/path/to/omnibus-toolchain/resources/omnibus-toolchain/ips/symlinks.erb"}
+    #
+    # @return [String]
+    #
+    def symlinks_file
```

Comment:
Incase both files exists we want to render the one with projectname, so added this method here :)!  Thank you for the helpful feedback.  I pushed the update addressing other review comments.